pkgs/development/haskell-modules/lib.nix

```diff
@@ -76,6 +76,10 @@ rec {
    */
   dontHaddock = drv: overrideCabal drv (drv: { doHaddock = false; });
 
+  /* setBuildTarget let's you specify which parts of the package to build
+   */
+  setBuildTarget = buildTarget: drv: overrideCabal drv (drv: { inherit buildTarget; });
```

Comment:
Two things here:

1. Could you add an example to this doc comment here showing what you might be able to set the `buildTarget` value to?  It would be nice if users could get an idea of what values they can set `buildTarget` to without having to look through the Cabal manual.
2. Could you add a test for `setBuildTarget`?  I'd like this just to make sure we don't accidentally break it at some point in the future.  You can add the test here: https://github.com/NixOS/nixpkgs/blob/afb7177ecada3a2ae341ed906656a0b3f8e9db0e/pkgs/test/default.nix.  Here's another Haskell test you could use as a base: https://github.com/NixOS/nixpkgs/blob/afb7177ecada3a2ae341ed906656a0b3f8e9db0e/pkgs/test/haskell-documentationTarball/default.nix.

Reply:
Done!